### PR TITLE
Migrate deploys to Fastlane

### PR DESCRIPTION
**Pull request by JBassett**
## Summary
Because our existing publishing plugin doesn't support the new Playstore tracks (https://github.com/Triple-T/gradle-play-publisher/issues/1088) we are going to move Fastlane (https://fastlane.tools/).  I've also added automated amazon app store publishing.  If we get into other app stores we should be able to use/make a plugin to deploy to that store as well.

## Screenshots

N/A

## Link to pull request in Documentation repository

N/A

### CODE REVIEW

File: fastlane/Fastfile [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+      skip_upload_screenshots: true,
+    )
+    supply(
+      track: 'wear:beta',
+      track_promote_to: 'wear:production',
+      skip_upload_changelogs: true,
+    )
+    supply(
+      track: 'automotive:beta',
+      track_promote_to: 'wear:production',
```
Comment (dshokouhi) [inline fastlane/Fastfile:74]: this should be `automotive:production` right?

File: fastlane/Fastfile [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+      track: 'wear:internal',
+      track_promote_to: 'wear:beta',
+      skip_upload_changelogs: true,
+      skip_upload_metadata: true,
+      skip_upload_images: true,
+      skip_upload_screenshots: true,
+    )
+    supply(
+      track: 'automotive:internal',
+      track_promote_to: 'wear:beta',
```
Comment (dshokouhi) [inline fastlane/Fastfile:49]: this should be `automotive:beta` right?

**Comment by JBassett** [comment]
Closed and reopened because Github decided my commit wasn't cool enough to showup.

**Review by dshokouhi — APPROVED** [review]
Looks good to me, thanks for getting this done before the Wear OS play console changes take effect.

**Pull request merged**